src/components/page-header/page-header-expandable.html.twig

```diff
@@ -35,13 +35,19 @@
 {% set _panel_id = panel_id|default('ecl-page-header-expandable-panel-' ~ random()) %}
 {% set _header_content = header_content|default('') %}
 {% set _panel_content = panel_content|default('') %}
+{% set _expandable = true %}
 {% set _separator = separator|e('html_attr')|default('–') %}
 {% set _extra_attributes = '' %}
 {% set _toggle_extra_attributes = toggle_extra_attributes|default([]) %}
 {% set _css_class = "ecl-page-header-expandable" %}
 
 {# Internal logic - Process properties #}
 
+{% if _lists is empty and _panel_content is empty %}
+  {% set _expandable = false %}
+  {% set _css_class =  _css_class ~ ' ecl-page-header-not-expandable' %}
```

Comment:
just a remark for the naming. We end up with things like `ecl-page-header-expandable ecl-page-header-not-expandable`, which is quite confusing. 
We may find something cleaner, using BEM, for instance `ecl-page-header-expandable--static`,  `ecl-page-header-expandable--simple`, `ecl-page-header-expandable--no-child`, ...